source/user/mp.rst

```diff
@@ -5,18 +5,134 @@ Second-Order Møller–Plesset Perturbation Theory
 
 *Modules*: :mod:`mp`, :mod:`pbc.mp`
 
+The interfaces to coupled cluster and MP2 calculations look very similar
+in PySCF.  Therefore, note the similarities of this section to the
+:ref:`theory_cc` documentation.
+
+
+Introduction
+============
+
+Second-Order Møller–Plesset Perturbation Theory (MP2) :cite:`Moller1934`
+is a post-Hartree--Fock method.
+
+A simple example (see :source:`examples/mp/00-simple_mp2.py`) of running
+an MP2 calculation is
+
+.. literalinclude:: ../../examples/mp/00-simple_mp2.py
+
+which outputs
+
+.. code::
+
+  converged SCF energy = -99.9873974403487
+  E(MP2) = -100.198764900659  E_corr = -0.211367460310054
+
+the Hartree--Fock energy, the MP2 energy and their difference, the
+MP2 correlation energy.
+
+.. note::
+
+  The last line in the code example above could have been replaced by
+
+  .. code ::
+    
+    pyscf.mp.MP2(mf).kernel()
+  
+  for the same result.
+
+
+Spin symmetry
+=============
+
+The MP2 module in PySCF supports a number of broken spin symmetry reference
+wavefunctions.  In particular, MP2 can be performed with a spin-restricted,
```

Comment:
This would be clearer as

"reference wave functions with broken spin symmetry"